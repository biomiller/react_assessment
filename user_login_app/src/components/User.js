import React from 'react';


export function User(props) {

  return (
    <div>
        <h3>{props.user.username}</h3>
    </div>

  );
}