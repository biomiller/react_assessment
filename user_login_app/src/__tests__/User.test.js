import React from 'react';
import renderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';

import { User } from "../components/User.js";


it('is User rendered', () => {

    const user={username:"Test"}

    const component = renderer
    .create(
      <MemoryRouter>
        <User user={user} />
      </MemoryRouter>
    )
    .toJSON();

    expect(component).toMatchSnapshot();
});



