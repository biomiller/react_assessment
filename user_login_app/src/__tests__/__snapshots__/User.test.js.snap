// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is User rendered 1`] = `
<div>
  <h3>
    Test
  </h3>
</div>
`;
